src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/LinkDefendantTest.java

```diff
@@ -10,51 +11,53 @@
 import uk.gov.hmcts.cmc.claimstore.tests.BaseTest;
 import uk.gov.hmcts.cmc.domain.models.Claim;
 
-import java.util.UUID;
-
-import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.cmc.ccd.assertion.Assertions.assertThat;
 
 public class LinkDefendantTest extends BaseTest {
 
+    private User claimant;
+
     @Autowired
     private FunctionalTestsUsers functionalTestsUsers;
 
+    @Before
+    public void before() {
+        claimant = idamTestService.createCitizen();
+    }
+
     @Test
     public void shouldBeAbleToSuccessfullyLinkDefendant() {
-        Claim createdCase = commonOperations.submitClaim(
-            functionalTestsUsers.getClaimant().getAuthorisation(),
-            functionalTestsUsers.getClaimant().getUserDetails().getId()
+        Claim claim = commonOperations.submitClaim(
+            claimant.getAuthorisation(),
+            claimant.getUserDetails().getId()
         );
 
-        User defendant = idamTestService.createCitizen();
+        User defendant = functionalTestsUsers.createDefendant(claimant.getUserDetails().getId());
 
-        Claim claim = linkDefendant(defendant, createdCase.getExternalId())
+        linkDefendant(defendant)
+            .then()
+            .statusCode(HttpStatus.OK.value());
+
+        Claim response = RestAssured
+            .given()
+            .header(HttpHeaders.AUTHORIZATION, defendant.getAuthorisation())
+            .when()
+            .get("/claims/" + claim.getExternalId())
             .then()
             .statusCode(HttpStatus.OK.value())
+            .assertThat()
```

Comment:
This assertion doesn't seem to be doing anything.

Reply:
fixed